lms/static/js/sysadmin/main.js

```diff
@@ -0,0 +1,25 @@
+function confirmAndSubmit(e){
+  var $target = $(e.target);
+  var args = $target.siblings("input").val();
+  var confirmMessage = 'Are you sure you want to execute the command: ' + $target.val() + ' with arguments: ' + args + ' ?';
+  if (confirm(confirmMessage)) {
+      var $form = $("#sysadmin-course-form");
+      var $hiddenInput = $(document.createElement("input"));
+      $hiddenInput.attr({
+        type: 'hidden',
+        name: 'action',
+      });
+      $hiddenInput.val($target.val());
+      $hiddenInput.appendTo($form);
+      $form.submit();
+  }
+}
+
+
+$(function(){
+    var insertTabButton = $('#insert-tab-button');
+    var deleteTabButton = $('#delete-tab-button');
```

Comment:
Also, do these actually need to be defined as variables?
Could you just do this:
```$('#insert-tab-button').bind("click", confirmAndSubmit);```

also, since we're in jQuery land:
```$('#insert-tab-button').click(confirmAndSubmit);```